syft/pkg/license_set.go

```diff
@@ -1,13 +1,13 @@
+//nolint:dupl
```

Comment:
why is this considered duplicated code? we should probably reuse existing functionality if needed